apps/challenges/admin.py

```diff
@@ -44,6 +61,71 @@ class ChallengeAdmin(ImportExportTimeStampedAdmin):
         "end_date",
     )
     search_fields = ("title", "creator__team_name", "slug")
+    actions = ["start_selected_workers", "stop_selected_workers", "scale_selected_workers", "restart_selected_workers", "delete_selected_workers"]
+    action_form = UpdateNumOfWorkersForm
+
+    def start_selected_workers(self, request, queryset):
+        response = start_workers(queryset)
+        count = response["count"]
+        message = response["message"]
```

Comment:
Can we please club the lines 69 and 70 by writing something like this:

> message, count = response["message"], response["count"]

Please make this change for other methods as well.